### PR TITLE
Minor fixes on image link and syntax enhancements

### PR DESCRIPTION
**Pull request by echarles**
I have done a quick pass to fix image links

I have also moved some images at the top of the README, and made a few syntax enhancements.

### CODE REVIEW

File: basics/hello-world/README.md [hunk 1]
```diff
@@ -191,16 +191,17 @@ Object { _started: true, _pluginMap: {…}, _serviceMap: Map(28), _delegate: {
 This is the main application JupyterLab object and we will see how to interact
 with it in the next sections.
 
-_checkout how the core packages of JupyterLab are defined at
-https://github.com/jupyterlab/jupyterlab/tree/master/packages . Each package is
+Checkout how the core packages of JupyterLab are defined
+[on this page](https://github.com/jupyterlab/jupyterlab/tree/master/packages). Each package is
 structured similarly to the extension that we are writing. This modular
-structure makes JupyterLab very adapatable_
+structure makes JupyterLab very adapatable
```
Comment (jtpio) [inline basics/hello-world/README.md]: Minor typo:

```suggestion
structure makes JupyterLab very adaptable.
```

File: basics/signals/README.md [hunk 2]
```diff
@@ -130,6 +131,4 @@ export class TutorialPanel extends StackedPanel {
 
 The final extension writes a little `changed` text to the browser console when
 a big red button is clicked. It is not very spectacular but the signaling is
-conceptualy important for building extensions. It looks like this:
-
-![Button with Signal](preview.png)
+conceptualy important for building extensions.
```
Comment (jtpio) [inline basics/signals/README.md]:
```suggestion
conceptually important for building extensions.
```

File: command-palette/README.md [hunk 3]
```diff
@@ -58,10 +61,8 @@ we want to inject into the `activate` function in the `JupyterFontEndPlugin`.
 category (that can be either an existing category or a new one). And optionally, you can specify
 the arguments that will be passed to the command when executed from the palette.
 
-When running JupyterLab with this extension, the following message should
-appears in the web browser console after clicking on the command in the palette:
-
-![Command Palette](../_images/command_palette.png)
+When running JupyterLab with this extension, a message should
+appears in the web browser console after clicking on the command in the palette.
```
Comment (jtpio) [inline command-palette/README.md]:
```suggestion
appear in the web browser console after clicking on the command in the palette.
```

File: widget-tracker/widgets/README.md [hunk 4]
```diff
@@ -1,12 +1,10 @@
-# Widgets
+# Widgets - Adding new Elements to the Main Window
 
-- [A basic tab](#a-basic-tab)
+We are going to do some real stuff and add a new tab to JupyterLab.
+Visible elements such as a tab are represented by widgets in the phosphor library
```
Comment (jtpio) [inline widget-tracker/widgets/README.md]:
```suggestion
Visible elements such as tabs and notebooks are represented by widgets in the Phosphor library
```

**Comment by jtpio** [comment]
Thanks @echarles.

We can take the opportunity to also fix a couple of typos from the previous iterations (comments).

**Comment by echarles** [comment]
Thx @jtpio Opportunities are there to be taken 👍 
I have use the easy button `commit suggestion` which is very easy for me but produces commit history.

We haven't discussed so far, but which kind of merge should we apply in general in this repo? Maybe it is a case by case. For this particular PR, I would be tempted to go for `Squash and merge`

**Comment by echarles** [comment]
Well, in this paritcular case, the advantage of keeping separated commits is to see the contributors

```
commit 741ae91fcb95a853f006813fd3a89911fe9fcce3
Author: Eric Charles <eric@datalayer.io>
Date:   Mon Dec 2 06:46:48 2019 -0600

    Update command-palette/README.md
    
    Co-Authored-By: Jeremy Tuloup <jeremy.tuloup@gmail.com>
```

**Comment by jtpio** [comment]
In normal cases we can go for the default merge strategy (no squash).

For this PR I think it's fine to squash and merge. Normally the contributors will all appear in the commit description.

**Comment by echarles** [comment]
Well github shows the contributors with multiple pictos in the commits view based, I guess, on the commit message having `Authored-by: ...` or `Signed-off-by: ...`

I leave you decide which option you choose clicking on the big green button.

**Comment by jtpio** [comment]
Let's go for "Squash And Merge" for this one, and we keep "Merge pull request" for the rest.

**Pull request merged**